### DISCUSSION TITLE
[Feature] multiple hearts/stars

### DISCUSSION THREAD

**Discussion by mmomjian** [immich-app/immich#14355] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be cool to have multiple hearts/stars/symbols, a la gmail. I'm picturing something like hovering over the heart symbol showing some more options. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Stars are already supported, you just need to enable them https://my.immich.app/user-settings?isOpen=feature+rating

**Comment by mmomjian** [comment]
Sorry, I don’t mean star ratings, I mean something like this: https://www.pcworld.com/article/426904/use-gmails-hidden-colored-stars-to-better-organize-important-emails.html

**Reply by bo0tzz** [reply]
Why?
